flask/app/__init__.py

```diff
@@ -34,10 +40,27 @@ def create_app(config):
     register_extensions(app)
     manage.register_commands(app)
     register_blueprints(app)
+    register_schedulers(app)
 
     return app
 
 
+def register_schedulers(app):
```

Comment:
We shouldn't be registering this altogether if we don't have an API key. We also definitely need an out-of-process cache as there are multiple Gunicorn workers, and we can't send out a batch of emails on exit because that would happen for every update we deploy and potentially spuriously since the server process could be restarted for whatever reason.